spaces/S000005/README.md

```diff
@@ -3,17 +3,18 @@ uid: S000005
 name: Odd-Even Topology
 counterexamples_id: 6
 refs:
-  - doi: 10.1007/978-1-4612-6290-9 
+  - zb: "0386.54001"
     name: Counterexamples in Topology
   - wikipedia: Partition_topology
     name: Partition topology on Wikipedia
 ---
-Define a topology on $\mathbb{N}$ by taking as a basis all sets of the form $\{\{2k-1,2k\}\mid k \in \mathbb{N}\}$.
+
+Define a topology on $X = \mathbb{N}$ by taking as a basis all sets of the form $\{\{2k-1,2k\}\mid k \in \mathbb{N}\}$.
```

Comment:
```suggestion
Define a topology on $X = \mathbb{N}$ by taking as a basis all pairs $\{2k-1,2k\}$ for $k \in \mathbb{N}$.
```
the previous phrasing was incorrect: "all sets of the form (the full collection here)" did not make sense.